packages/react_rearch/lib/react_rearch.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:meta/meta.dart';
+import 'package:rearch/rearch.dart';
+
+export 'src/side_effects.dart';
+export 'src/widgets.dart';
+
+/// The API exposed to [RearchConsumer]s to extend their functionality.
+///
+/// New methods may be added to this interface on any new _minor_ release
+/// (minor in terms of semver).
+@experimental
+abstract interface class ComponentSideEffectApi implements SideEffectApi {
+  /// Adds a deactivate lifecycle listener.
+  // void addDeactivateListener(SideEffectApiCallback callback);
+
+  /// Removes the specified deactivate lifecycle listener.
+  // void removeDeactivateListener(SideEffectApiCallback callback);
```

Comment:
Could you please remove these commented out parts if there is no React-specific equivalent?